docs/quickstart.md

```diff
@@ -57,67 +57,90 @@ docker run -it \
 
 
 ```bash
-kubectl port-forward service/facebook-opt-125m-modelserver 8080:8080
+kubectl port-forward service/falcon-7b-instruct-server 8080:8080
 ```
 
-All substratus ModelServers ship with an API and interactive frontend. Open up your browser to [http://localhost:8080/](http://localhost:8080/) and talk to your model! Alternatively, request text generation via the HTTP API:
+All substratus Servers ship with an API and interactive frontend. Open up your browser to [http://localhost:8080/](http://localhost:8080/) and talk to your model! Alternatively, request text generation via the OpenAI compatible HTTP API:
 
 
 ```bash
  curl http://localhost:8080/v1/completions \
   -H "Content-Type: application/json" \
   -d '{ \
-    "model": "facebook-opt-125m", \
-    "prompt": "The quick brown fox ", \
-    "max_tokens": 30, \
-    "temperature": 0 \
-  }'
-  # choices[0].text will very different
-{"id":"cmpl-2d4c8871b20dc45e6ac98322","object":"text_completion","created":1688628294,"model":"facebook-opt-125m","choices":[{"text":"I've read Patrick Beut, Richard Eichel, Elliot Gagné","index":0,"logprobs":null,"finish_reason":"length"}],"usage":{"prompt_tokens":1,"completion_tokens":16,"total_tokens":17}}
+    "model": "falcon-7b-instruct", \
+    "prompt": "Who was the first president of the United States? ", \
+    "max_tokens": 12\
+  }' | jq
 ```
 
-If you are interested in continuing your journey through Substratus, take a look at the [Guided Walkthrough](./category/walkthrough).
+      % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                     Dload  Upload   Total   Spent    Left  Speed
+      0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100   458  100   326  100   132    171     69  0:00:01  0:00:01 --:--:--   241
+    [1;39m{
+      [0m[34;1m"id"[0m[1;39m: [0m[0;32m"cmpl-e42772faf58cd46c18a955f1"[0m[1;39m,
+      [0m[34;1m"object"[0m[1;39m: [0m[0;32m"text_completion"[0m[1;39m,
+      [0m[34;1m"created"[0m[1;39m: [0m[0;39m1689485483[0m[1;39m,
+      [0m[34;1m"model"[0m[1;39m: [0m[0;32m"falcon-7b-instruct"[0m[1;39m,
+      [0m[34;1m"choices"[0m[1;39m: [0m[1;39m[
+        [1;39m{
+          [0m[34;1m"text"[0m[1;39m: [0m[0;32m"\nGeorge Washington was the first president of the United States."[0m[1;39m,
+          [0m[34;1m"index"[0m[1;39m: [0m[0;39m0[0m[1;39m,
+          [0m[34;1m"logprobs"[0m[1;39m: [0m[1;30mnull[0m[1;39m,
+          [0m[34;1m"finish_reason"[0m[1;39m: [0m[0;32m"length"[0m[1;39m
+        [1;39m}[0m[1;39m
+      [1;39m][0m[1;39m,
+      [0m[34;1m"usage"[0m[1;39m: [0m[1;39m{
+        [0m[34;1m"prompt_tokens"[0m[1;39m: [0m[0;39m11[0m[1;39m,
+        [0m[34;1m"completion_tokens"[0m[1;39m: [0m[0;39m12[0m[1;39m,
+        [0m[34;1m"total_tokens"[0m[1;39m: [0m[0;39m23[0m[1;39m
+      [1;39m}[0m[1;39m
+    [1;39m}[0m
```

Comment:
Do these color characters work on the actual site? Github doesn't render them properly.

Reply:
Nope it doesn't render them :(
![image](https://github.com/substratusai/substratusai.github.io/assets/388784/09381015-496d-4229-adad-5823a5c3c62b)

Reply:
it looked so nice in my vscode

Reply:
why can't everything look as nice as vscode? 😭